README.md

```diff
@@ -148,5 +150,9 @@ Current brownie version has problems with coverage reports for some contracts. C
 - [RemoveRewardProgram.sol](https://github.com/lidofinance/easy-track/blob/a72858804481009f2e09508ffbf93d8a4aee6c84/contracts/EVMScriptFactories/RemoveRewardProgram.sol#L23)
 - [TopUpLegoProgram.sol](https://github.com/lidofinance/easy-track/blob/a72858804481009f2e09508ffbf93d8a4aee6c84/contracts/EVMScriptFactories/TopUpLegoProgram.sol#L26)
 - [TopUpRewardProgram.sol](https://github.com/lidofinance/easy-track/blob/a72858804481009f2e09508ffbf93d8a4aee6c84/contracts/EVMScriptFactories/TopUpRewardPrograms.sol#L27)
+- [AddAllowedRecipient.sol](TBA)
```

Comment:
because I thought, that lines will move and I'll forget to fix the links.

Because of the last comments to remove immutable variables, I removed everything except TopUp factory and added link for it 😄